### ISSUE TITLE
p-multiSelect, p-inputNumber, p-treeTable Accessibility Issues

### ISSUE THREAD

**Issue by srujana10499** [primefaces/primeng#18771]
### Describe the bug

The p-multiSelect, p-inputNumber, p-treeTable components has accessibility violations when running the IBM Equal Accessibility Checker. The issue is: aria specification violations

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/stackblitz-starters-2eg7tx8m

### Environment

Stackblitz

### Angular version

20.0.4

### PrimeNG version

v20

### Steps to reproduce the behavior

1. Install the [IBM Equal Accessibility Checker](https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp)
2. Open the stackblitz
3. Run the accessibility checker
4. Filter on violations only and note the errors: 
     a. Form control with "combobox" role has no associated label
     b. Form control with "spinbutton" role has no associated label
     c. The ARIA attributes "aria-level, aria-checked" are not valid for the element <tr> with implicit ARIA role "row"

### Expected behavior

There are no accessibility violations for the p-multiSelect, p-inputNumber, p-treeTable components
